canonical/ubuntu.com
# WD-7815 - redirect from embedded to real-time

## Done

Added redirect from /embedded to /real-time page

## QA

- [demo link](https://ubuntu-com-13391.demos.haus/embedded)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that /embedded page is redirected to /real-time

## Issue / Card
[WD-7729](https://warthogs.atlassian.net/browse/WD-7729)

Fixes #

## Screenshots

[WD-7729]: https://warthogs.atlassian.net/browse/WD-7729?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
